react/Trigger.tsx

```diff
@@ -35,8 +36,17 @@ function Trigger(props: Props) {
     }
   }, [dispatch, containerRef])
 
+  useEffect(() => {
+    if (dispatch) {
```

Comment:
Why is that needed?

Reply:
Add a comment here

Reply:
`useContext` returns `T | undefined`

Reply:
I changed the type to always return a function